### PR TITLE
Bypass CORS check in igv.js using cors-anywhere service and add IGV.js tracks documentation

### PR DESCRIPTION
**Pull request by northwestwitch**
Allow igv.js to use any cloud resource, even if the cloud storage provider doesn't support the strict igv.js CORS policy.

**How to test**:
1. Add this track configuration to the demo config file:
**Note that the chromosome build is "38"**. This way you could test the PR by looking at one MT variant in IGV (uses chromosome build 38)

```
CLOUD_IGV_TRACKS = [
    {
        "name": "custom_public_bucket",
        "access": "public",
        "tracks": [
            {
                "name": "dbVar Pathogenic or Likely Pathogenic",
                "type": "variant",
                "format": "vcf",
                "build": "38",
                "url": "https://s3-eu-west-1.amazonaws.com/pfigshare-u-files/25777457/GRCh37.variant_call.clinical.pathogenic_or_likely_pathogenic.vcf.gz",
                "indexURL": "https://s3-eu-west-1.amazonaws.com/pfigshare-u-files/25777460/GRCh37.variant_call.clinical.pathogenic_or_likely_pathogenic.vcf.gz.tbi",
            }
        ],
    },
]
```

2. from master branch, launch demo server (`scout --demo serve`) and go to this MT variant: http://localhost:5000/cust000/643594/f75deb14514df16195735a4466eae2d0
3. Click on the igv browser (make sure dbVar track is selected)
4. Make sure that the dbVar track doesn't load
5. Switch to this branch and repeat
6. Now the dbVar track should be shown

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
Found a solution @dnil!

**Comment by northwestwitch** [comment]
Master branch:

image


This branch:

image

File: docs/admin-guide/igv-settings.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+- Reference genome track
+- Genes track
+- ClinVar SNVs track
+- ClinVar CNVs track
+
+Reference genome and genes track are collected from the [Broad Institute Amazon S3 storage][amazon_s3], while the ClinVar tracks are available in the UCSC Genomics Institute [Sequence and Annotation Downloads][ucsc_downloads]
+
+
+### Custom tracks
```
Comment (dnil) [inline docs/admin-guide/igv-settings.md:15]: :+1:

File: scout/server/blueprints/alignviewers/templates/alignviewers/igv_viewer.html [hunk 2]
```diff
@@ -32,6 +32,8 @@
   <div class="container-fluid" id="igvDiv" style="padding:5px; border:1px solid lightgray"></div>
 </body>
 
+{% set cors_anywhere_prefix = "https://cors-anywhere.herokuapp.com" -%}
```
Comment (dnil) [inline scout/server/blueprints/alignviewers/templates/alignviewers/igv_viewer.html:35]: As long as the tracks are not secret in any way this looks ok! I guess we will depend on the proxy being up as well, but there are other dependencies as well. We could use this now, and keep looking for a local code solution perhaps?

**Review by dnil — APPROVED** [review]

File: scout/server/blueprints/alignviewers/templates/alignviewers/igv_viewer.html (hunk 2 above)
Comment (northwestwitch) [reply scout/server/blueprints/alignviewers/templates/alignviewers/igv_viewer.html:35]: The page looks widely used but you are right, you never know! The good thing is that if it stops working then our igv browser won't crash, only the track will display error!

**Comment by northwestwitch** [comment]
Merging this one! Thanks for the review ©dnil!

**Pull request merged**